src/components/teampage/advisors/advisor-member.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { AdvisorCard, AdvisorImg } from "./advisor-member-styles"
+import { MemberName, ConnectLink, MemberInfo } from "../team-shared-styles"
+
+export default ({ name, photo, position, linkedinUrl }) => (
+  <AdvisorCard>
+    <AdvisorImg fluid={photo.fluid} alt={photo.description} />
```

Comment:
Oh wait, all the advisors seem different sizes on mobile?
<img src="https://user-images.githubusercontent.com/60420625/117220882-22008c80-add6-11eb-8314-cb89dc38c4b7.jpg" alt="advisors" width="300" />

Reply:
🤢

Reply:
😪